frontendUiTableImpl/src/main/java/com/virtuslab/gitmachete/frontend/ui/impl/table/GitMacheteGraphTableManager.java

```diff
@@ -197,36 +197,47 @@ public void run(ProgressIndicator indicator) {
    * Updates repository which is the base of graph table model. The change will be seen after
    * {@link GitMacheteGraphTableManager#refreshGraphTable()}.
    */
-  @SuppressWarnings({"IllegalCatch"})
+  @SuppressWarnings({"IllegalCatch", "subtyping"})
```

Comment:
Hmm why `subtyping` suppression is necessary here? :thinking:

Reply:
Coz I wasn't be able to do `cause.getCause()` without this

Reply:
Hmm and what was the error? :hushed:

Reply:
Ok, I'm looking into that now, possibly some defaults are missing in `SubtypingTop`

Reply:
Ok, fixed, just remove `@SuppressWarnings` completely here and remove `@DefaultFor` from `SubtypingBottom`:

```diff
diff --git a/frontendUiTableImpl/src/main/java/com/virtuslab/gitmachete/frontend/ui/impl/table/GitMacheteGraphTableManager.java b/frontendUiTableImpl/src/main/java/com/virtuslab/gitmachete/frontend/ui/impl/table/GitMacheteGraphTableManager.java
index d77cd3d6..93e32517 100644
--- a/frontendUiTableImpl/src/main/java/com/virtuslab/gitmachete/frontend/ui/impl/table/GitMacheteGraphTableManager.java
+++ b/frontendUiTableImpl/src/main/java/com/virtuslab/gitmachete/frontend/ui/impl/table/GitMacheteGraphTableManager.java
@@ -197,7 +197,6 @@ public final class GitMacheteGraphTableManager implements IGraphTableManager {
    * Updates repository which is the base of graph table model. The change will be seen after
    * {@link GitMacheteGraphTableManager#refreshGraphTable()}.
    */
-  @SuppressWarnings({"IllegalCatch", "subtyping"})
   private void updateRepository(Path mainDirectoryPath, Path gitDirectoryPath, boolean isMacheteFilePresent) {
     LOG.debug(() -> "Entering: mainDirectoryPath = ${mainDirectoryPath}, gitDirectoryPath = ${gitDirectoryPath}" +
         "isMacheteFilePresent = ${isMacheteFilePresent}");
diff --git a/qual/src/main/java/com/virtuslab/qual/internal/SubtypingBottom.java b/qual/src/main/java/com/virtuslab/qual/internal/SubtypingBottom.java
index 118454ba..d179961f 100644
--- a/qual/src/main/java/com/virtuslab/qual/internal/SubtypingBottom.java
+++ b/qual/src/main/java/com/virtuslab/qual/internal/SubtypingBottom.java
@@ -3,7 +3,6 @@ package com.virtuslab.qual.internal;
 import java.lang.annotation.ElementType;
 import java.lang.annotation.Target;
 
-import org.checkerframework.framework.qual.DefaultFor;
 import org.checkerframework.framework.qual.SubtypeOf;
 import org.checkerframework.framework.qual.TargetLocations;
 import org.checkerframework.framework.qual.TypeUseLocation;
@@ -28,5 +27,4 @@ import com.virtuslab.qual.gitmachete.frontend.graph.api.items.ConfirmedCommitIte
     ConfirmedBranchItem.class,
     ConfirmedCommitItem.class,
 })
-@DefaultFor(TypeUseLocation.LOWER_BOUND)
 public @interface SubtypingBottom {}
```

Reply:
Apparently, with that default, Checker was assuming that `Consumer`'s type param is always `@SubtypingBottom`, which doesn't make much sense

Reply:
But actually... there's also a different style problem here - I'd rather not re-assign anything to lambda param, preferably make a separate var for the use in `while`

Reply:
It seems to work  :tada: